test/com/google/enterprise/adaptor/database/JdbcFixture.java

```diff
@@ -47,15 +72,63 @@
    * The default escape character is disabled, to match the DQL
    * behavior (and standard SQL).
    */
+  static {
+    Database dbname = Database.H2;
+    String dbdriver = "org.h2.Driver";
+    String dburl = "jdbc:h2:mem:test;DB_CLOSE_DELAY=-1;DEFAULT_ESCAPE=";
+    String dbuser = "sa";
+    String dbpassword = "";
+
+    String propertiesFile = System.getProperty("build.properties");
+    if (!Strings.isNullOrEmpty(propertiesFile)) {
+      try (BufferedReader br =
+          Files.newBufferedReader(
+              FileSystems.getDefault().getPath(propertiesFile),
+              Charset.forName("UTF-8"))) {
+        Properties properties = new Properties();
+        properties.load(br);
+        if (!Strings
+            .isNullOrEmpty(properties.getProperty("test.jdbc.database"))) {
+          String name = properties.getProperty("test.jdbc.database");
+          if (name.equalsIgnoreCase(Database.SQLSERVER.toString())) {
```

Comment:
dbname = Database.valueOf(name.toUpperCase());
and drop the tags and toString() method of the enum.
and catch IllegalArgumentException along with FNFException to use defaults.